sklearn/tree/_classes.py

```diff
@@ -244,9 +244,9 @@ def fit(self, X, y, sample_weight=None, check_input=True,
             elif self.max_features == "log2":
                 max_features = max(1, int(np.log2(self.n_features_)))
             else:
-                raise ValueError(
-                    'Invalid value for max_features. Allowed string '
-                    'values are "auto", "sqrt" or "log2".')
+                raise ValueError("Invalid value for max_features. "
+                                 "Allowed string values are 'auto', "
+                                 "'sqrt' or 'log2'.")
```

Comment:
@alfaro96 as a side not, please do not include such unrelated stylistic changes in your PRs, it makes them more likely to cause conflicts when dealing with back ports for maintenance branches. Better keep the PR focused on the issue of the description.